piker/clearing/_ems.py

```diff
@@ -578,11 +588,9 @@ async def translate_and_relay_brokerd_events(
 
             relay.positions.setdefault(
                 # NOTE: translate to a FQSN!
-                f'{sym}.{broker}',
-                {}
-            ).setdefault(
-                pos_msg['account'], {}
-            ).update(pos_msg)
+                (broker, sym),
```

Comment:
So instead of a table of tables we just do a flat table with a `tuple[str, str]` of the broker and account name.